src/components/PostList/PostList.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './PostList.scss';
+import { PostInfo } from '../PostInfo/PostInfo';
+
+type Props = {
+  posts: Post[],
+};
+
+export const PostList: React.FC<Props> = ({ posts = [] }) => (
```

Comment:
prop posts is required
```suggestion
export const PostList: React.FC<Props> = ({ posts }) => (
```